samples/contractupdate/package.json

```diff
@@ -4,7 +4,6 @@
     "private": true,
     "dependencies": {
         "@concordium/react-components": "0.1.0",
-        "@concordium/wallet-connectors": "0.1.0",
```

Comment:
How come this is removed? Isn't there still a reference to this is `src/Root.tsx`?

Reply:
The library is being reexported via `react-components` so shouldn't be used directly. This change really belonged with https://github.com/Concordium/concordium-dapp-libraries/commit/f571d59feee65581ea1750b3126aec4389281a5a.